internal/inbox/channel/telegram/telegram.go

```
@@ -0,0 +1,563 @@
+
+	if msg.Sticker != nil {
+		// For animated (.tgs) or video (.webm) stickers, use the thumbnail if available.
+		// Static stickers (.webp) can be downloaded directly.
+		if msg.Sticker.IsAnimated || msg.Sticker.IsVideo {
+			if msg.Sticker.Thumbnail != nil {
+				if att, err := t.downloadFile(msg.Sticker.Thumbnail.FileID, "sticker.jpg", "image/jpeg"); err == nil {
+					attachments = append(attachments, att)
+				} else {
+					t.lo.Error("error downloading telegram sticker thumbnail", "error", err)
+				}
+			} else {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)